app/views/shared/email/_submission_block.html.erb

```diff
@@ -8,8 +8,8 @@
               <table class="contents">
                 <tr>
                   <td>
-                    <% if @submission.processed_images.length > 0 %>
-                      <%= image_tag(@submission.processed_images.first.image_urls['square'], id: 'first-image') %>
+                    <% if submission.processed_images.length > 0 %>
+                      <%= image_tag(submission.processed_images.first.image_urls['square'], id: 'first-image') %>
```

Comment:
Updating this template to use the passed arguments `artist` and `submission` instead of the global ones.